test/support/utils_test_base.h

```diff
@@ -319,14 +319,20 @@ struct test_base
 
 namespace
```

Comment:
you can remove this unnamed `namespace` because it stops being helpful here

Reply:
Fixed